### PR TITLE
Fixes #32497 - allow template to properly manage vlan tag over bond interface

### PR DESCRIPTION
**Pull request by jpasqualetto**
Allow the template kickstart_kernel_options to create the correct set of
options for dracut, in order to provision a server using a tagged VLAN over a bond
interface.

### CODE REVIEW

File: app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb
Comment (lzap) [inline app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb:85]: Yeah this is the only reasonable way to find if an interface is attached to a bond. We do have `bond?` method but there is no direct database association among these. :-(

**Review by lzap — DISMISSED** [review]
LGTM

**Comment by lzap** [comment]
Thank you for your contribution, let's wait until tests pass and I will merge it. Takes 1-2 hours :-)

**Comment by lzap** [comment]
Can you rebase just to trigger new commit and tests? There was a problem on Redmine it was assigned with a wrong project.

**Review by lzap — APPROVED** [review]
LGTM just waiting for tests to pass. I think we do not have a bond snapshot template test but I am not sure.

**Comment by tbrisker** [comment]
ok to test
(otherwise they won't run)

**Review by lzap — APPROVED** [review]
TY

**Comment by ezr-ondrej** [comment]
CP 2.5: #8587

**Pull request merged**
